po/extra/it.po

```diff
@@ -0,0 +1,62 @@
+# French translations for extra package.
+# Copyright (C) 2019 THE extra'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the extra package.
+# NathanBnm, 2019.
```

Comment:
Just a small thing: This comment mentions "French translations" and "NathanBnm".

Same for the other file.